keras/engine/training.py

```diff
@@ -1969,7 +1969,7 @@ def fit_generator(self,
                       use_multiprocessing=False,
                       shuffle=True,
                       initial_epoch=0):
-        """Trains the model on data yielded batch-by-batch by a Python generator.
+        """Trains the model on data generated batch-by-batch by a Python generator.
```

Comment:
Sequence are not generators. So maybe 'Data generator' would be more suitable?

Reply:
I'd like to preserve phrasing "Python generator". Maybe "Python generator or an instance of `Sequence`"? That also corresponds to the docstring for the `generator` parameter:
>        # Arguments
>            generator: A generator or an instance of `Sequence`

https://github.com/keras-team/keras/blob/master/keras/engine/training.py#L1976

Reply:
Sounds good